manifests/x/Xanthus58/NetworkPoolCalculatorReforged/1.1.0/Xanthus58.NetworkPoolCalculatorReforged.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Xanthus58.NetworkPoolCalculatorReforged
+PackageVersion: 1.1.0

```

```suggestion
PackageVersion: 1.0.0
```
This must match Control Panel to prevent upgrade loops while running `winget upgrade Xanthus58.NetworkPoolCalculatorReforged`